### DISCUSSION TITLE
Iframe Issues

### DISCUSSION THREAD

**Discussion by Pranav-P-16** [darkreader/darkreader#6919] [Q&A]
I've noticed that darkreader extension converts Iframes to dark mode, but while embedding darkreader using unpkg it's not acting on Iframes, Does anyone know the reason for that ?
The code is
```
<script src="https://unpkg.com/darkreader@4.9.35/darkreader.js"></script>
<script>
DarkReader.enable({
    brightness: 100,
    contrast: 90,
    sepia: 10
});
</script>
```

The Result is (The background colour is converted to black from white but nothing happens on iframe :disappointed: )
![issue1](https://user-images.githubusercontent.com/91425738/134816091-72ca63c7-76ea-47ba-aba2-d9d9c07fa253.png)

But when using Browser Extension, The result is (Everything fine on iframe, the white background doesn't bother as background changes when used on website)
![issue2](https://user-images.githubusercontent.com/91425738/134816154-16206d80-84fd-4a27-9ace-2c1186ead58c.png)

**Comment by Gusted** [comment]
Please do not open up discussions for just opened issues.
The Issue: #6918
